dashboard/components/RelationDependencyGraph.tsx

```diff
@@ -150,31 +164,63 @@ export default function RelationDependencyGraph({
         .attr("font-family", "inherit")
         .attr("text-anchor", "middle")
         .attr("dy", nodeRadius * 2)
-        .attr("fill", "black")
         .attr("font-size", 12)
         .attr("transform", "rotate(-8)")
 
+      // Relation type
+      let typeText = g.select<SVGTextElement>(".type")
+      if (typeText.empty()) {
+        typeText = g.append("text").attr("class", "type")
+      }
+
+      const relationTypeAbbr = (relation: Relation) => {
+        const type = relationType(relation)
+        if (type === "SINK") {
+          return "K"
+        } else {
+          return type.charAt(0)
+        }
+      }
```

Comment:
Can you add a legend or sth? I too often feel confused about the meanings of abbrs 😟

Reply:
Add a tooltip for that! 😄